packages/synapse-react-client/src/components/ForumSearch/ForumSearch.tsx

```diff
@@ -125,57 +121,61 @@ export const ForumSearch = (props: ForumSearchProps) => {
             }
           }}
         />
-        {searchInput && (
-          <button
-            className="ClearSearchIcon"
-            onClick={() => {
-              onResetSearch()
-            }}
-          >
-            <IconSvg icon="clear" />
-          </button>
-        )}
       </div>
-      {noSearchResult && (
+      {isShowingResults && (
         <>
-          {props.projectId && (
-            <Typography variant="body1Italic" className="NoResultsText">
-              No results for &quot;{searchInput}&quot; in {entity?.name}
-            </Typography>
+          {noSearchResult && (
+            <>
+              {props.projectId && entity && (
+                <Typography variant="body1Italic" className="NoResultsText">
+                  No results for &quot;{lastSearchTerm}&quot; in {entity?.name}
```

Comment:
Fix bug where the text would sync with the current input state, when it should match the last query.